sdk/azidentity/live_test.go

```diff
@@ -56,12 +57,42 @@ var liveUser = struct {
 }
 
 const (
-	fakeClientID   = "fake-client-id"
-	fakeResourceID = "/fake/resource/ID"
-	fakeTenantID   = "fake-tenant"
-	fakeUsername   = "fake@user"
+	fakeClientID      = "fake-client-id"
+	fakeResourceID    = "/fake/resource/ID"
+	fakeTenantID      = "fake-tenant"
+	fakeUsername      = "fake@user"
+	fakeAdfsAuthority = "fake.adfs.local"
+	fakeAdfsScope     = "fake.adfs.local/fake-scope/.default"
 )
 
+var adfsAuthority = os.Getenv("ADFS_AUTHORITY_HOST")
+
+var adfsLiveSP = struct {
+	tenantID    string
+	clientID    string
+	secret      string
+	pemPath     string
+	scope       string
+	redirectURL string
+}{
+	tenantID:    "adfs",
+	clientID:    os.Getenv("ADFS_SP_CLIENT_ID"),
+	secret:      os.Getenv("ADFS_SP_CLIENT_SECRET"),
+	pemPath:     os.Getenv("ADFS_SP_CERT_PEM"),
+	scope:       os.Getenv("ADFS_SCOPE"),
```

Comment:
I'm thinking this should be a package-level variable (`var adfsScope = os.Getenv("ADFS_SCOPE")`) or duplicated on `adfsfLiveUser`

Reply:
Changed to package level